assets/sounds/extinct2.ogg.import

```diff
@@ -0,0 +1,15 @@
+[remap]
+
+importer="ogg_vorbis"
+type="AudioStreamOGGVorbis"
+path="res://.import/extinct2.ogg-6122e9d17446baa4f9ae52089e919650.oggstr"
+
+[deps]
+
+source_file="res://assets/sounds/extinct2.ogg"
+dest_files=[ "res://.import/extinct2.ogg-6122e9d17446baa4f9ae52089e919650.oggstr" ]
+
+[params]
+
+loop=true
```

Comment:
Shouldn't be looped.

Reply:
Ah yes, that's a good catch. 
This is why we need to store the .import files in the repo @adQuid ...

Reply:
I fixed this but I won't bump the CI caches, because so far this is only used in one music group where it is the only song and loops anyway. I did re-import this on my system so this'll be entirely correct in the releases.